src/ControllerV2/RosalieInteroperabilityController.php

```diff
@@ -63,6 +63,18 @@ private function processSiSiaoNumberForm(Request $request, Beneficiaire $benefic
         ]);
     }
 
+    private function processBeneficiaryCheckResult(BeneficiaryCheckOnRosalie $beneficiaryCheck): void
```

Comment:
I'm not sure this method is necessary if it is only used once. But if we want to create a method anyway, maybe we could change its name ? It could look more like this (not sure about the name) : 
```
private function checkBeneficiaryOnRosalie(Beneficiary $beneficiary): void
{
    $beneficiaryCheck = $this->service->checkBeneficiaryOnRosalie($beneficiary); 
    .... etc
}
```